content/realtime/channels/channel-parameters/deltas.textile

```diff
@@ -0,0 +1,207 @@
+
+When subscribing without an Ably library, then the channel @delta@ parameter must be specified using a "qualified channel name":/realtime/channels/channel-parameters/overview. In the case of SSE, it is also possible to specify channel parameters as regular query parameters on the connection URL.
+
+Some transports provide raw message payloads - that is, the content of the @data@ attribute of a @Message@ - without the accompanying metadata. That means that the recipient of the message does not have access to the @extras@ or @encoding@ attributes of the message that would ordinarily be used to decode delta message payloads. 
+
+Examples of such transports are MQTT, and SSE in non-enveloped mode. In order to assist applications that use these transports, the @vcdiff@ decoder libraries can check for the @vcdiff@ magic number at the start of the message payload as an inexact method of determining whether or not the message is a regular message or a delta. Note that, in order to rely on that check, you need to know that that magic number will not be present in any valid (uncompressed) message in your app. No valid JSON value, for example, will match the @vcdiff@ header check, so it is safe to perform this sniffing on JSON message payloads.
+
+h4(#delta-example-sse). Delta example with SSE
+
+You can subscribe to messages in delta mode, using the SSE transport, as follows.
+
+```
+(() => {
+    const key = '{{API_KEY}}';
+    const channel = 'sample-app-sse';
+    const url = `https://realtime.ably.io/event-stream?channels=${channel}&v=1.1&key=${key}&delta=vcdiff`;
+    const eventSource = new EventSource(url);
+    const channelDecoder = new DeltaCodec.CheckedVcdiffDecoder();
+
+    eventSource.onmessage = (event) => {
+        /* event.data is JSON-encoded Ably Message (see https://www.ably.io/documentation/realtime/types#message) */
+        const message = JSON.parse(event.data);
+        const { id, extras } = message;
+        let { data } = message;
+
+        try {
+            if (extras && extras.delta) {
+                data = channelDecoder.applyBase64Delta(data, id, extras.delta.from).asUtf8String();
+            } else {
+                channelDecoder.setBase(data, id);
+            }
+        } catch(e) {
+            /* Delta decoder error */
+            console.log(e);
+        }
+
+        /* Process decoded data */
+        console.log(data);
+    };
+})();
+```
+
+h4(#delta-example-unenv-sse). Delta example with unenveloped SSE
+
+```(() => {
+    const key = '{{API_KEY}}';
+    const channel = 'sample-app-sse';
+    const url = `https://realtime.ably.io/event-stream?channels=${channel}&v=1.1&key=${key}&delta=vcdiff&enveloped=false`;
+    const eventSource = new EventSource(url);
+    const channelDecoder = new DeltaCodec.VcdiffDecoder();
+
+    eventSource.onmessage = (event) => {
+        let data = event.data;
```

Comment:
In the previous example there was a ` /* event.data is JSON-encoded Ably Message (see https://www.ably.io/documentation/realtime/types#message) */` here.  Why is this inconsistent?

Reply:
@paddybyers / @QuintinWillison would this just be the same link? Also is this just the same language?